src/mutation/RelayMutationQuery.js

```diff
@@ -242,6 +243,21 @@ var RelayMutationQuery = {
         }
       }
     }
+
+    // Warn if the newly created edge was not included in mutatedFields
+    const edgeField = mutatedFields.find(field => {
```

Comment:
How about adding an `else` clause at [line 225](https://github.com/facebook/relay/pull/574/files#diff-e8b6d06b88b10658255cc62756b5c59eR225)? `mutatedEdgeFields` is only blank if no range behaviors matched and the new edge field would be missing.